app/requirements/requirements.txt

```diff
@@ -113,32 +113,32 @@ django==4.2.15
 django-appconf==1.0.6
     # via django-celery-email
 django-celery-beat==2.6.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-celery-email==3.0.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-celery-results==2.5.1
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-cors-headers==4.4.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-extensions==3.2.3
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-filter==24.2
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-fsm==3.0.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-markdownx==4.0.7
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-o365mail==1.1.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-silk==5.1.0
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-storages==1.14.4
-    # via -r requirements.in
+    # via -r requirements/requirements.in
 django-timezone-field==7.0
     # via django-celery-beat
 djangorestframework==3.15.2
     # via
-    #   -r requirements.in
+    #   -r requirements/requirements.in
```

Comment:
Als je het pip commando draait vanuit de requirements map, dan worden niet al deze changes gedaan. Dat is misschien overzichtelijker.